NEWS

```diff
@@ -25,6 +27,8 @@ Standard:
   . Partly fix GH-12143 (Incorrect round() result for 0.49999999999999994).
     (timwolla)
   . Fix GH-12252 (round(): Validate the rounding mode). (timwolla)
+  . Added 4 new rounding modes to the round(). (Jorg Sowa)
```

Comment:
You should not be updating NEWS - this is updated by person merging the PR together with UPGRADING.

Reply:
I was told in my other PRs to update `UPGRADING` file. So I see there are different opinions on this and I'm not sure now which should I follow.

If I update the UPGRADING file in the same PR isn't it easier for the person merging PR? This saves time for them.

Reply:
Not sure who told you that but it's usually done by commiter because those files (especially NEWS) might change often so it might lead to conflicts. It's not a big issue for master only change and UPGRADING as they don't change that often but it's still usually done by commiter at the time of the commit.

Reply:
It is easier to have the committer of the PR to update UPGRADING then push this task on the person merging the PR.

Especially as UPGRADING is only ever touched in master.

But NEWS, AFAIK, is only ever used to inform about bug fixes.

Reply:
Apparently, and I've only learned this only recently, you should update both UPGRADING and NEWS.

The only case when UPGRADING is not updated is for bug fixes in release branches.